build.sh

```diff
@@ -218,50 +218,20 @@ command_test() {
     local test_file="test_python_binary_dist.py"
     local python_binary="$PYTHON_BIN"
 
+    echo "::group::Chevah tests"
     if [ ! -d build/ ]; then
         (>&2 echo "No build/ sub-directory present, try 'build' first!")
         exit 220
     fi
+
+    echo "#### Executing Chevah shell tests... ####"
+    source src/chevah-bash-tests/shellcheck_tests.sh
```

Comment:
this is ok.
I was expecting to have it called from the github actions yaml file.

but if you prefer this, I am ok.

Reply:
Sorry, the commented diff is outdated. That script is no longer sourced, it works independently.

But it's true I don't want it to be something tied to GHA because it's mighty useful to have this available before committing. Thanks!

Reply:
Current output:
```
$ ./src/chevah-bash-tests/shellcheck_tests.sh    
## Getting shellcheck binary in build/ if missing... ##
Shellcheck already installed.
## Checking executable .sh files in the root dir of the repo... ##
Executable shell scripts to be checked (including their sources):
        ./build.sh
        ./publish_dist.sh
        ./pythia.sh
Non-executable scripts to be skipped (should be sourced by the above):
        ./functions.sh
        ./functions_build.sh
        ./os_quirks.sh
        ./pkg_checks.sh
## Checking extra .sh files in "./src/"... ##
Extra shell scripts to be checked under ./src (including sources):
        ./src/chevah-bash-tests/get-shellcheck.sh
        ./src/chevah-bash-tests/shellcheck_tests.sh
        ./src/chevah-python-tests/get_binaries_deps.sh
## Checking the chevahbs scripts in ./src/*/ sub-dirs in their dirs... ##
chevahbs files to be checked were found in the following sub-dirs:
        ./src/Python
        ./src/Python-Windows
        ./src/bzip2
        ./src/libedit
        ./src/libffi
        ./src/openssl
        ./src/sqlite-autoconf
        ./src/xz
        ./src/zlib
```